### DISCUSSION TITLE
我运行了run bat提示报错

### DISCUSSION THREAD

**Discussion by xiahao775** [GaiZhenbiao/ChuanhuChatGPT#986] [Q&A]
我的pip版本已经升级到21.3.1
![图片](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/51218227/654965a6-b591-42e4-8b58-7d436fba945b)

**Comment by GaiZhenbiao** [comment]
你在使用什么版本的python呢？请使用较新版本的Python，例如3.10

**Reply by xiahao775** [reply]
用的3.10版本
